### PR TITLE
Switch updater to systemd system service

### PR DESCRIPTION
**Pull request by pluehne**
This makes the `systemd` system service a user service. There are two reasons that motivated this change.

1. User services require an active session of the specified user. This has the drawback that the updater would not be triggered after reboots if no SSH connection to the GitHub Enterprise appliance was made (see also #71, which should be fixed by this pull request).
1. User services cannot depend on system services. Because of that, the updater service could not reliably wait for the rest of GitHub Enterprise to start.

### CODE REVIEW

File: updater/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ The updater may be run in two modes:
 
 The updater’s log is accessible as follows:
 ```sh
-journalctl -f --user-unit hubble-enterprise.service
+journalctl -fu hubble-enterprise.service
```
Comment (larsxschneider) [inline updater/README.md]: this change is unrelated, right?

File: updater/packaging/debian/templates/Makefile [hunk 2]
```diff
@@ -2,7 +2,7 @@ all: # nothing to build
 
 PKGNAME = hubble-enterprise
 INSTALLDIR = $(DESTDIR)/opt/autodesk/$(PKGNAME)
-SYSTEMDDIR = $(DESTDIR)/home/admin/.local/share/systemd/user
+SYSTEMDDIR = $(DESTDIR)/usr/lib/systemd/system
```
Comment (larsxschneider) [inline updater/packaging/debian/templates/Makefile]: Do you see a way to move our stuff into a subdirectory here? Would that make sense?

File: updater/README.md (hunk 1 above)
Comment (pluehne) [reply updater/README.md]: No, it isn’t. `-u` stands for `--unit` and refers to system units, while `--user-unit` is the equivalent for user units and doesn’t have a corresponding shorthand option.

File: updater/packaging/debian/templates/Makefile (hunk 2 above)
Comment (pluehne) [reply updater/packaging/debian/templates/Makefile]: I think that `systemd` doesn’t look for unit files in a subdirectory.
